pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -0,0 +1,629 @@
+// api.pb.go could be generate by hack/update-generate-runtime.sh
+syntax = 'proto2';
```

Comment:
As long as there are no significant issue for upgrading to proto3, I am fine with proto2 :)
Thanks for checking!